dev/bin/interaction/delete-test-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
+
+# Script used to delete Bridges and Processors used in E2E tests
+# Script will delete Bridges without Processors, for Bridges with Processors it will delete Processors if they are in proper state
+
+BRIDGES=($(curl -s -H "Authorization: $OB_TOKEN" -X GET $MANAGER_URL/api/v1/bridges | jq '.items[] | select(.name | startswith("test-")) | .id' | tr -d \"))
```

Comment:
I would suggest to move "test-" prefix to a separate variable at the beginning of the script. It will be more clear how the script finds out bridges. Something like BRIDGE_PREFIX="test-".

Reply:
Unfortunately it is not so easy as the jq filter is specified using single quote, making properly replacement complicated. I have added some comments mentioning "test-" filtering in name.